src/unix/bsd/netbsdlike/openbsd/x86_64.rs

```diff
@@ -24,39 +24,80 @@ s_no_extra_traits! {
 }
 
 cfg_if! {
-    if #[cfg(feature = "extra_traits")] {
-        impl PartialEq for fxsave64 {
-            fn eq(&self, other: &fxsave64) -> bool {
-                self.fx_fcw == other.fx_fcw &&
-                self.fx_fsw == other.fx_fsw &&
-                self.fx_ftw == other.fx_ftw &&
-                self.fx_fop == other.fx_fop &&
-                self.fx_rip == other.fx_rip &&
-                self.fx_rdp == other.fx_rdp &&
-                self.fx_mxcsr == other.fx_mxcsr &&
-                self.fx_mxcsr_mask == other.fx_mxcsr_mask &&
-                self.fx_st == other.fx_st &&
-                std::ptr::addr_of!(self.fx_xmm) == std::ptr::addr_of!(other.fx_xmm)
+    if #[cfg(feature = "extra_traits", feature = "libc_addr_of")] {
```

Comment:
```suggestion
    if #[cfg(feature = "extra_traits", libc_addr_of)] {
```

It isn't a feature gate.